### DISCUSSION TITLE
react-next use videojs-landscape-fullscreen warning

### DISCUSSION THREAD

**Discussion by lvqi0724** [vercel/next.js#20460] [Help] [answered]
import videojs from "video.js";

//  Error using this plug-in Cannot set property 'lockOrientationUniversal' of undefined
import 'videojs-landscape-fullscreen'

const [videoNode, setVideoNode] = useState();
    const [player, setPlayer] = useState();
  
    useMemo(() => {
      ....//
        if (videoNode) {            
            const videoPlayer = videojs(videoNode, videoJsOptions);
            videoPlayer.landscapeFullscreen({
                enterOnRotate: true,
                alwaysInLandscapeMode: true,
                iOS: true
            })
            setPlayer(videoPlayer);
        }
    }, [videoNode]);

    useEffect(() => {
        return (() => {
            if (player) player.dispose()
        })
    }, [])

**Comment by lvqi0724** [comment]
Baby please help me

**Comment by naayaghan27** [comment]
😊

**Comment by lvqi0724** [comment, marked as answer]
A solution has been found

see:

player.jsx

import videojs from 'video.js'
import 'videojs-landscape-fullscreen'

....
videoPlayer.landscapeFullscreen({
                fullscreen: {
                    enterOnRotate: true,
                    alwaysInLandscapeMode: true,
                    iOS: true
                }
            })

--------------------------------------------

import dynamic from 'next/dynamic'
const Player = dynamic(
    () => import('../../components/player'),
    { ssr: false }
  )
